README.md

```diff
@@ -35,9 +35,20 @@
 
 ## Releasing OpenSearch
 
+### About OpenSearch releases and versions
+
+The OpenSearch Project releases as versioned bundles of OpenSearch, OpenSearch Dashboards, and the OpenSearch plugins. The OpenSearch Project [follows semantic versioning](https://opensearch.org/blog/technical-post/2021/08/what-is-semver/). Software, such as Data Prepper, clients, and the Logstash output plugin, are versioned independently of the OpenSearch Project. They also may have independent releases from the main project distributions. In addition to the semantic versioning, the OpenSearch Project may release software under alpha, beta, release candidate, and generally available labels. The definition of when to use these labels is derived from [the Wikipedia page on Software release lifecycle](https://en.wikipedia.org/wiki/Software_release_life_cycle). Below is the definition of when to use each label.
+
+Release labels:
+
+* **Alpha** - The code is released with instructions to build. Built distributions of the software may not be available. Some features many not be complete. Additional testing and developement work is planned. Distributions will be postfixed with `-alpha.X` where "X" is the number of the alpha version.
+* **Beta** - Built distributions of the software are available. All features are completed. Additional testing and developement work is planned. Distributions will be postfixed with `-beta.X` where "X" is the number of the beta version.
+* **Release Candidate** - Built distributions of the software are available. All features are completed. Code is tested and minimal validation remains. At this stage the software is potentially stable and will release unless signficant bugs emerge. Distributions will be postfixed with `-rc.X` where "X" is the number of the release candidate version.
+* **Generally Available** - Built distributions of the software are available. All features are completed and documented. All testing is completed. Distributions for generally available versions are **NOT** postfixed with an additional label.
```

Comment:
I would remove the bold/CAPITALIZED **NOT**, feels shouty.

Reply:
ack.